src/types/untyped.d.ts

```diff
@@ -1,3 +0,0 @@
-// Plain JS libs
-declare module '@cfpb/cfpb-expandables/src/Expandable';
-declare module '@cfpb/cfpb-forms';
```

Comment:
These definitions were created to get rid of ESLint errors but I find that they block the ability to navigate to the actual component definitions in `node_modules`.  Having that navigation enabled  feels more useful than resolving the ESLint error.